eclipse-collections-code-generator/src/main/resources/api/collection/mutablePrimitiveCollection.stg

```diff
@@ -51,7 +51,8 @@ public interface Mutable<name>Collection extends <name>Iterable
     default boolean removeIf(<name>Predicate predicate)
     {
         boolean changed = false;
-        for (Mutable<name>Iterator iterator = this.<type>Iterator(); iterator.hasNext();)
+        Mutable<name>Iterator iterator = this.<type>Iterator();
+        while (iterator.hasNext())
```

Comment:
Why this change?

Reply:
It is for one of the checkstyle fixes. I can revert and add a space after the ; in the precious version. But logic wise it'll do the same thing or am I missing something

Reply:
ok